### PR TITLE
Mip examples

### PR DESCRIPTION
**Pull request by dhendryc**
Added a folder with some examples from the MIP lib and their corresponding mps files.

### CODE REVIEW

**Comment by dhendryc** [comment]
If a point does not quite satisfy our tolerances in `is_linear_feasible` we print a warning with the distance from the point and set. Only if the point is a unit away from the set, `false` is returned and the asserts break.

File: src/interface.jl [hunk 1]
```diff
@@ -79,7 +104,7 @@ function solve(
         @assert !MOI.is_valid(lmo.o, cidx)
     end
 
-    direction = ones(n)
+    direction = rand(n)
```
Comment (matbesancon) [inline src/interface.jl:107]: rand like this makes the solve function non-deterministic, hence the one instead

File: src/callbacks.jl [hunk 2]
```diff
@@ -6,8 +6,16 @@ function build_FW_callback(tree, min_number_lower, check_rounding_value::Bool, f
     return function fw_callback(state, active_set)
         @assert isapprox(sum(active_set.weights), 1.0)
         @assert sum(active_set.weights .< 0) == 0
-        @assert is_linear_feasible(tree.root.problem.lmo, state.x)
         @assert is_linear_feasible(tree.root.problem.lmo, state.v)
+        @assert is_linear_feasible(tree.root.problem.lmo, state.x)
+       #= if !is_linear_feasible(tree.root.problem.lmo, state.v)
```
Comment (matbesancon) [inline src/callbacks.jl:11]: let's cleanup the commented code before merging

File: src/node.jl [hunk 3]
```diff
@@ -109,6 +109,11 @@ function Bonobo.evaluate_node!(tree::Bonobo.BnBTree, node::FrankWolfeNode)
         tree.root.problem.integer_variables,
     )
 
+    # At the root node activate reoptimization for SCIP
+    #if MOI.get(tree.root.problem.lmo.lmo.o, MOI.SolverName()) == "SCIP"
```
Comment (matbesancon) [inline src/node.jl:113]: same here

**Pull request merged**